_posts/2019-11-12-docker-rename-to-container.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Pulp Docker plugin will be renamed to Pulp Container plugin in Pulp3
+author: Ina Panova
+tags:
+  - docker
+  - container
+---
+
+With Pulp 3 GA pulp_container 1.0 will come along.
```

Comment:
Version 1.0 of the pulp_container plugin will be released to coincide with Pulp 3 GA.